### DISCUSSION TITLE
gl_nearest

### DISCUSSION THREAD

**Discussion by pozzurrecconiolicini** [Novum/vkQuake#746] [Q&A] [answered]
There doesn't seem to be a way to use a `gl_texturemode` like `gl_nearest` with a cvar. It seems like texture filtering is handled differently with Vulkan... Are there any plans to support something like `gl_texturemode gl_nearest`? Or maybe there is a way to use it that I'm just not aware of? If not, how technically difficult would it be to implement that type of feature?

**Comment by vsonnier** [comment, marked as answer]
What you seek is the setting `Options > Graphics > Textures > classic / smooth`. The corresponding CVAR is `vid_filter`.

**Reply by pozzurrecconiolicini** [reply]
Yeah, I know about that setting. It isn't quite the same as `gl_nearest`, as I do see some "blurriness" in the textures from far away (which I can decrease with supersample antialiasing, although it has a significant performance impact on my end). It isn't a huge issue though. I was just curious about that, since I prefer the more pixelated look of `gl_nearest`.
